timesketch/frontend-ng/src/components/Explore/AggregateDialog.vue

```diff
@@ -0,0 +1,537 @@
+<!--
+Copyright 2023 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <v-card class="mx-auto">
+    <v-card-title>
+      <span class="headline">Event Data Analytics</span>
+      <v-spacer></v-spacer>
+      <v-btn icon @click="clearAndCancel">
+        <v-icon>mdi-close</v-icon>
+      </v-btn>
+    </v-card-title>
```

Comment:
In other dialogs (see the graph bottom sheet for example) we use a v-toolbar. This will make sure that the toolbar is visible when scrolling etc.

```
        <v-toolbar dense flat>
          <strong>Title</strong>
          <v-spacer></v-spacer>
          <v-btn icon @click="showTimelineView = false">
            <v-icon>mdi-close</v-icon>
          </v-btn>
        </v-toolbar>
        ```

Reply:
I found a solution to ensure the toolbar is visible when scrolling.

Reply:
Cool solution with the toolbar that stays on-top when scrolling 👍  
However, now the field & value information are missing that have been in the subtitle before. I personally liked it, since it gave an immediate overview what field and value I'm looking at. Maybe you have a good idea on how to bring it back into the toolbar? What do you think?